examples/patterns/table-mobile-card/table-mobile-card.html

```diff
@@ -0,0 +1,50 @@
+---
+layout: default
+title: Table mobile card
+category: _patterns
+---
+
+<table class="p-table--mobile-card" role="grid">
+  <thead>
+    <tr role="row">
+      <th scope="col" role="columnheader" id="t-name" aria-sort="none">Name</th>
+      <th scope="col" role="columnheader" id="t-users" aria-sort="none" class="u-align--right">Users</th>
+      <th scope="col" role="columnheader" id="t-units" aria-sort="none" class="u-align--right">Units</th>
+      <th scope="col" role="columnheader" id="t-revenue" aria-sort="none" class="u-align--right">Actions</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr role="row">
+      <td role="rowheader" aria-label="Name">Grape</td>
+      <td role="gridcell" aria-label="Users"class="u-align--right">8</td>
+      <td role="gridcell" aria-label="Units" class="u-align--right">19</td>
+      <td role="gridcell" class="u-align--right">
+        <span class="p-contextual-menu p-contextual-menu--left" role="menubar">
```

Comment:
It not a good idea to nest patterns if you can help it. If the pattern changes to create multiple places to update the example. Feel free to document the usage if you feel it's useful.